lib/commons/aria/is-accessible-ref.js

```diff
@@ -49,17 +16,10 @@ function isAccessibleRef(node) {
 	// because axe.commons is not available in axe.utils, we can't do
 	// this caching when we build up the virtual tree
 	if (!cache.get('idRefs')) {
-		cache.set('idRefs', {});
-		// Get all idref(s) attributes on the lookup table
-		const refAttrs = Object.keys(standards.ariaAttrs).filter(attr => {
-			const { type } = standards.ariaAttrs[attr];
-			return idRefsRegex.test(type);
-		});
-
-		cacheIdRefs(root, refAttrs);
+		cacheIdRefs(root);
 	}
 
-	return cache.get('idRefs')[id] === true;
+	return !!cache.get('idRefs')[id];
```

Comment:
Would be better for this to call `getAccessibleRef()` instead of reaching into its cache.